src/NodeApi/Native/JSNativeApi.Interop.cs

```diff
@@ -1323,31 +1319,31 @@ internal static napi_status napi_detach_arraybuffer(napi_env env, napi_value arr
         }
 
         internal static napi_status napi_object_freeze(napi_env env, napi_value js_object)
-            => CallInterop(FunctionId.napi_object_freeze, env, js_object.Handle);
+            => CallInterop(ref s_fields.napi_object_freeze, env, js_object.Handle);
 
         internal static napi_status napi_object_seal(napi_env env, napi_value js_object)
-            => CallInterop(FunctionId.napi_object_seal, env, js_object.Handle);
+            => CallInterop(ref s_fields.napi_object_seal, env, js_object.Handle);
 
-        private static nint GetExport(FunctionId functionId, [CallerMemberName] string functionName = "")
+        private static nint GetExport(ref nint field, [CallerMemberName] string functionName = "")
```

Comment:
Is there a reason to pass the fields by ref when they are just `nint`?

Reply:
Oh I see, because the field may be intialized on first use.

Reply:
This function also initializes the passed fields. This is why we need it to be an in-out parameter.